themes/bootstrap3/js/validation.js

```diff
@@ -0,0 +1,66 @@
+/* global VuFind, isPhoneNumberValid */
+
+VuFind.register('validation', function Validation() {
+  /**
+   * Check field match validity
+   * @param {Event} ev Event
+   * @returns {boolean} Validity
+   */
+  function checkMatchValidity(ev) {
+    const field = ev.target;
+    const matchSelector = field.dataset.match;
+    if (!matchSelector) {
+      return true;
+    }
+    const matchEl = field.form.querySelector(matchSelector);
+    if (matchEl.value !== field.value) {
+      field.setCustomValidity(field.dataset.matchError || '');
+      return false;
+    }
+    field.setCustomValidity('');
+    return true;
+  }
+
+  /**
+   * Check field phone number validity
+   * @param {Event} ev Event
+   * @returns {boolean} Validity
+   */
+  function checkPhoneNumberValidity(ev) {
+    const field = ev.target;
+    if (field.id && field.type === 'tel' && field.dataset.validatorRegion) {
+      const valid = isPhoneNumberValid(field.value, field.dataset.validatorRegion);
```

Comment:
If I'm following things correctly, I believe that isPhoneNumberValid is being exported by libphonenumber. It looks to me like we're using a version of libphonenumber from 2010 or so, which is probably outdated and inaccurate. It's out of scope for the current PR, but maybe when we update this for the 11.0 release, we should also introduce loading of libphonenumber via NPM (possibly using the lighter-weight [libphonenumber-js](https://www.npmjs.com/package/libphonenumber-js)) and make sure things work with the latest version. What do you think?

Reply:
That all would make sense, but I'd handle it separately. Not sure if [libphonenumber-js](https://www.npmjs.com/package/libphonenumber-js) supports usage without a module system, but any update would probably be better than the current one.

Reply:
Agreed, definitely a separate issue, but worth doing before we forget about it. I can open a JIRA ticket if you think that would be the best next step.

Reply:
Please do open a ticket!

Reply:
Done: [VUFIND-1751](https://openlibraryfoundation.atlassian.net/browse/VUFIND-1751)